app/javascript/app/components/ghg-emissions/ghg-emissions-selectors/ghg-emissions-selectors-filters.js

```diff
@@ -115,23 +115,40 @@ const getBreakByOptions = () =>
       }
     ]);
 
+// Filtered calculation selectors
+const getFilteredCalculationOptions = createSelector(
+  [getCalculationOptions, getSourceSelected],
+  (calculationOptions, sourceSelected) => {
+    if (!calculationOptions || !sourceSelected) return null;
+    if (sourceSelected.name === 'UNFCCC_NAI') {
+      return calculationOptions.filter(({ value }) => value !== 'CUMULATIVE');
```

Comment:
Could we use the constant here as: CALCULATION_OPTIONS.CUMULATIVE.value ?

Reply:
Good catch! Fixed and ready to merge!

Reply:
There are going to be some conflicts with my next PR